### PR TITLE
USWDS jekyll integration

### PR DESCRIPTION
**Pull request by jordangov**
This PR pulls in the US Web Design Standards jekyll plugin and uses its config options for the header and footer. No real content changes here.

Thoughts welcome... I'll merge in a couple days if there's no comments. (Note: this is still a WIP and is not a deployed merge, just a merge into the WIP branch `static-site`.)

### CODE REVIEW

**Comment by jgarber623-gov** [comment]
Site builds fine locally without issues. 👍

File: Gemfile.lock [hunk 1]
```diff
@@ -14,18 +21,18 @@ GEM
...
+    ethon (0.11.0)
       ffi (>= 1.3.0)
     execjs (2.7.0)
     faraday (0.13.1)
       multipart-post (>= 1.2, < 3)
     ffi (1.9.18)
     forwardable-extended (2.6.0)
     gemoji (3.0.0)
-    github-pages (164)
+    github-pages (168)
```
Comment (nschonni) [inline Gemfile.lock]: It appears your removing this from the Gemfile, but still have it in the Lockfile

Comment (jordangov) [reply Gemfile.lock]: Ack! good call, I think I didn't run `bundle` fresh after that removal.

**Comment by jbjonesjr** [comment]
Speaking of the USWDS theme, wanted to make sure you all saw this:

https://github.com/github/pages-gem/pull/495

This allows you to use the theme from it's remote host instead of having to manually add it to your site.

**Comment by jordangov** [comment]
That's neat! Looks like they have an issue with GH ("GitHub does not provide support for Jekyll Remote Theme")... not sure if that means GH pages or what. That said, having the theme locally, with a locked down version, actually helps with testing on local dev machines - for example while I'm programming without wifi (planes and such).

**Comment by jbjonesjr** [comment]
> GitHub does not provide support

> support

Basically ben is just noting this is our early access, it's use at your own risk for the next few days before we genpop it. Plan is to make it available to all this calendar year.

:+1: to advantages of a local copy

**Comment by jordangov** [comment]
Ooooh... gotcha. Yeah... words matter. My bad.

**Pull request merged**